torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1411,6 +1411,73 @@ def sample_inputs_linalg_norm(op_info, device, dtype, requires_grad):
                             dim=(0, 1))))
         return inputs
 
+def sample_inputs_as_strided(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(torch.tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    # input shape, output shape, output stride, output storage offset
+    test_cases = [
```

Comment:
This is cool but maybe a few more samples, like the examples in the function's documentation, would be interesting, too?